README.md

```diff
@@ -7,65 +7,86 @@ This getting started will show you how to work with MapR-DB JSON using:
 * MapR-DB Shell to interact with JSON tables from the command line
 * Apache Drill to so some analytics with SQL
 * Java and the OJAI library to build operational application.
+* Spark-DB connector to natively load / store RDDs to table.
 
 You will learn how to optimize query using MapR-DB Secondary indexes.
 
 **Prerequisites**
 
-* MapR Converged Data Platform 6.0 Beta with Apache Drill
+* MapR Converged Data Platform 6.0 Beta with Apache Drill OR [MapR Developer Sandbox Container](https://maprdocs.mapr.com/home/MapRContainerDevelopers/MapRContainerDevelopersOverview.html).
 * JDK 8
 * Maven 3.x
 
+## Setting up MapR Developer Sandbox Container
 
+MapR Developer Sandbox Container is an effort to get MapR running on your developer machine so that one could quickly get in to working within MapR environment.
+
```

Comment:
I would not use the word "effort".  That makes it seem like it's going to be hard to get MapR up and running.  Alternatives:

"MapR Developer Sandbox Container is an offering that enables you to quickly deploy a MapR environment on your developer machine".

OR

"MapR Developer Sandbox Container is a docker image that enables you to quickly deploy a MapR environment on your developer machine."

Reply:
Agreed. Will make this change.